### ISSUE TITLE
ibm.power_aix.user causes Ansible to crash (regression?)

### ISSUE THREAD

**Issue by stephenulmer** [IBM/ansible-power-aix#145]
**Describe the bug**
The following task has worked for months with the current ibm.aix_power collection release:

```
- name: Change the root User's Home and Limits
  ibm.power_aix.user:
    name: root
    attributes:
      home: /root
      fsize: -1
      data: -1
      rss: -1
      stack: -1
      nofiles: -1
    state: modify
  register: result
```

**Expected behavior**
The module should modify the user's attributes (in the released version of the collection this module is *not* idempotent, but that is not this bug).

**Result**
```
fatal: [as1]: FAILED! => changed=false 
  module_stderr: |-
    Shared connection to 10.103.41.54 closed.
  module_stdout: |-
    Traceback (most recent call last):
      File "/root/.ansible/tmp/ansible-tmp-1629839046.1022058-250307-75807398015697/AnsiballZ_user.py", line 102, in <module>
        _ansiballz_main()
      File "/root/.ansible/tmp/ansible-tmp-1629839046.1022058-250307-75807398015697/AnsiballZ_user.py", line 94, in _ansiballz_main
        invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
      File "/root/.ansible/tmp/ansible-tmp-1629839046.1022058-250307-75807398015697/AnsiballZ_user.py", line 40, in invoke_module
        runpy.run_module(mod_name='ansible_collections.ibm.power_aix.plugins.modules.user', init_globals=None, run_name='__main__', alter_sys=True)
      File "/opt/freeware/lib/python2.7/runpy.py", line 176, in run_module
        fname, loader, pkg_name)
      File "/opt/freeware/lib/python2.7/runpy.py", line 82, in _run_module_code
        mod_name, mod_fname, mod_loader, pkg_name)
      File "/opt/freeware/lib/python2.7/runpy.py", line 72, in _run_code
        exec code in run_globals
      File "/tmp/ansible_ibm.power_aix.user_payload_v3lTDk/ansible_ibm.power_aix.user_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/user.py", line 338, in <module>
  
      File "/tmp/ansible_ibm.power_aix.user_payload_v3lTDk/ansible_ibm.power_aix.user_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/user.py", line 327, in main
  
      File "/tmp/ansible_ibm.power_aix.user_payload_v3lTDk/ansible_ibm.power_aix.user_payload.zip/ansible_collections/ibm/power_aix/plugins/modules/user.py", line 143, in modify_user
  
      File "/opt/freeware/lib/python2.7/re.py", line 141, in match
        return _compile(pattern, flags).match(string)
    TypeError: expected string or buffer
  msg: |-
    MODULE FAILURE
    See stdout/stderr for the exact error
  rc: 1
```
**Environment (please complete the following information):**
 - OS: AIX 7.2 TL5 SP1
 - Python Version: 2.7 (from AIX Toolbox)
 - AIX Collection Version: dev-collection HEAD

**Additional context**
The task is used inside of a role.

**Comment by pvtorres** [comment]
HI Stephen,
Indeed. It seems a fix that we did for another issue broke the user.py if a value is not pass as a string. Workaround, surround the values with quotes.
I will post a fix today for review today.

**Comment by pvtorres** [comment]
I fixed it in #148  . Thanks Stephen
